mllib/src/test/scala/org/apache/spark/mllib/rdd/MLPairRDDFunctionsSuite.scala

```diff
@@ -22,9 +22,13 @@ import org.apache.spark.mllib.rdd.MLPairRDDFunctions._
 import org.apache.spark.mllib.util.MLlibTestSparkContext
 
 class MLPairRDDFunctionsSuite extends SparkFunSuite with MLlibTestSparkContext {
+  val source_array = Array(
```

Comment:
Also, I think we use naming convention, `sourceArray`. Probably, just `data` is enough?